### ISSUE TITLE
New site design for flarerpg.org

### ISSUE THREAD

**Issue by clintbellanger** [clintbellanger/flare-engine#1407]
The current Flare website is old and junky, it's time for a new one.
- Clean neutral design, not based on the alpha demo
- Modern standards (responsive design)
- A new focus. We're mostly past that early internal rough engineering phase. Now it's time to focus on modders and community.
- The site should mainly be about downloads and documentation for the engine.
- The alpha demo, fantasycore content shouldn't be the default front and center game in screenshots and videos
- The dev blog should be reworked, and posting privileges should be available to all the main devs (probably the same ones with merge rights in flare-engine)
- Add presskit()

I'm completely fine with using an existing solution like Wordpress if that makes the most sense. Also completely fine with using a free CSS theme as a starting point.

Here are other sites in our Free/Libre community that I like, and I think Flare Engine can have a similar feel.
- Tiled http://www.mapeditor.org/ incredibly clean, delicate use of shadows and depth, neutral colors to let the images speak for themselves, responsive design that resizes elegantly
- FIFE engine http://www.fifengine.net/ simple, clean, responsive, geared towards engine users, gets right to the point

If any experienced web designers out there want to help the Flare project in this way, please let me know.

**Comment by clintbellanger** [comment]
Related note. On the current site, I replaced the "archive" blog link with a new one, better suited for the amount of content shown there. http://flarerpg.org/blog/archive/ generated from the RSS feed.

**Comment by dorkster** [comment]
I've recently done a bit of experimenting with a few CMSes in an effort to prototype my ideas for the new site. I tried Concrete5, Drupal, and Wordpress. I ended up picking Wordpress, since it seems to handle what we want in the most user friendly fashion.

I'm wondering if I'm on the right track design-wise. Here are some screenshots of what I've put together so far:

![site1](https://cloud.githubusercontent.com/assets/913982/11018332/fdd85fe8-8592-11e5-9458-d77e50e041d5.png)
![site2](https://cloud.githubusercontent.com/assets/913982/11018334/fdd9502e-8592-11e5-9e97-251706ef7581.png)
![site3](https://cloud.githubusercontent.com/assets/913982/11018333/fdd8d66c-8592-11e5-81b2-019641414fe0.png)

**Comment by clintbellanger** [comment]
I think this looks great. Nice and clean.

Agreed that Wordpress has the features the page needs.

**Comment by dorkster** [comment]
Today, I worked on some scripts to migrate all 325 blog posts over to the new site: [(screenshot of the main blog page)](http://i.imgur.com/w4p2qrN.jpg)

All images (and links to images) should work. Of course there are some dead links in the older posts, but most of those are dead on the current site as well.

**Comment by stefanbeller** [comment]
Looks awesome!

Though I find it slightly hard to read, can you increase the blackness of regular text a little bit?

**Comment by dorkster** [comment]
Imgur converted my screenshot to a JPG, so that washed some of the colors out. Either way, the theme I used is pretty low contrast.

Does this look better?:
![blog flare 2015-12-21 23-58-55](https://cloud.githubusercontent.com/assets/913982/11948173/d80ae096-a83e-11e5-9e68-ffcc9ec841ef.png)

**Comment by stefanbeller** [comment]
I really like that better. :)

(Unrelated?) side note: I just watched https://media.ccc.de/v/32c3-7547-libusb_maintainer_fail#video
('How I failed maintaining an Open Source Project') and one of the major issues the guy pointed out was very occasional releases. From the outside the releases are the 'heartbeats' which let's you tell if the project is alive. Do you plan on releasing any time soon? (I am so out of the loop I cannot tell the current quality of the code, etc)

**Comment by clintbellanger** [comment]
I like the way the template looks. Also, if you want a raw dump of the Flare mysql database I can provide that.

I'm about ready to get back in the loop myself for the project. I'll be looking for things to work on to help towards the 1.0 launch.

**Comment by dorkster** [comment]
For anyone that's *really* out of the loop, I wrote a [blog post](http://dorkster.github.io/blog_2015-12-29_21-42-55.html) summarizing the activity around Flare for the last year and a half.

For those that are caught up, here's what I've been up to for the last day or so:

----

I roped some of my friends into play-testing the latest engine + Empyrean mod. The feedback has been good so far. It's led to some UI fixes, and some experimentation with how the action queue works.

Also from the play-testing, I've gather that the game is hitting a good spot in terms of difficulty. At least up to the first dungeon. There were some deaths, but players were always able to progress soon after.

Despite working on some engine things and banging my head against #1434, I'm attempting to shift myself into full content creation mode. My goal is to complete 1 map a day, though I expect some maps to take 2 days. A "map", in this case, includes the map itself, any quests attached to it, and the enemies that inhabit it.

So far, so good. I completed a [map + side-quest combo today](https://github.com/clintbellanger/flare-game/commit/48f6e2e93b7c2693b3dc1b69aed9fe45857486b5). Thanks to some new
[automapping rules](https://github.com/clintbellanger/flare-game/commit/2f4cce859404c9e435780d801294ac9cc72be72e), I was able to work at a faster pace by cutting out the time spent drawing in collision tiles.

![2016-01-03_223809](https://cloud.githubusercontent.com/assets/913982/12083337/3c38c1f2-b26f-11e5-8ab1-e19a76c3242b.png)

I will definitely create such rules for the other tilesets. It's just that it seemed more worthwhile with the snowplains tileset as it has so many objects.
